neocore/IO/BinaryReader.py

```diff
@@ -223,8 +223,8 @@ def ReadVarInt(self, max=sys.maxsize):
             int:
         """
         fb = self.ReadByte()
-        if fb is None:
-            return 0
+        if fb is 0:
```

Comment:
because `self.ReadByte()` never returns `None` but actually `0`.

Reply:
`if not fb` might also work